cmd/security-secrets-setup/Dockerfile

```diff
@@ -86,4 +86,6 @@ RUN chmod 755 security-secrets-setup
 
 ENTRYPOINT ["entrypoint.sh"]
 
+STOPSIGNAL SIGTERM
```

Comment:
FYI: This gives exit code `143` if docker-compose down.  Not sure if any blackbox testing is checking the exit code of services or not.

Reply:
Not sure.  This was blocking blackbox testing and I haven't gotten all the way through getting blackbox testing to see if this is of consequence.

Reply:
I don't know if this is desirable.  `STOPSIGNAL SIGTERM` sets the signal used to terminate the container.  On receipt, this would collapse the container.  I suspect it would be better to allow the signal to pass through to the application.  If the application is using the common bootstrap, the signal should result in a graceful shutdown.

Reply:
Not quite right.  STOPSIGNAL is the signal that is set to a docker container when `docker stop` (or `docker-compose down`) is issued.  Actually the default STOPSIGNAL is already SIGTERM so I am going to remove it.

Reply:
(steady state of the container is that it is hung on /bin/tail after setting up PKI.)